components/Feature/VulnerabilitiesGrid/grid.json

```diff
@@ -1,269 +1,269 @@
 [
   {
-    "id":"financial-stability",
-    "name":"Financial stability",
-    "assets":[
+    "id": "financial-stability",
+    "name": "Financial stability",
+    "assets": [
       {
-        "label":"Stable job"
+        "label": "Stable job"
       },
       {
-        "label":"Other",
-        "textinputs":[
+        "label": "Other",
+        "textinputs": [
           {
-            "label":""
+            "label": "Other"
```

Comment:
Why have these labels been added to the inputs as well as the checkboxes?

Reply:
Maybe we could come up with something better here - it feels strange having Other -> Other

Reply:
Because when storing an object, these labels are being used as keys and can't be empty:
_vulnerabilities: [{name: "Other", data: {Other: "whatever was entered"}}]_
It's not very necessary in this case, but when there are multiple text boxes for one check box I think it'll make sense, like:
_vulnerabilities: [{name: "Contact", data: {phone: "whatever was entered", email: "something else"}}]_

Reply:
Hm, can think of something else, in the summary page it would get displayed as:
Vulnerabilities:
* vuln1
* vuln2
* Other:
    - whatever was entered

Reply:
The fact that there are multiple items makes it a good case for using an array. Another good reason is if we need to add extend these items we're going have to change the structure again, ie value will need to go from being a string to being another object

Reply:
I see, I'm not entirely convinced if objects are harder to extend, could just nest them (?), I guess it's not as flexible, so I'm okay with changing this to an array, I'll try that now

Reply:
Processing nested stuff like that is gonna start looking gross no? I'd say as a general rule groups of objects work best as an array. That said, being able to look things up by key will probably make your add/remove code simpler